lib/app/console_app.rb

```diff
@@ -151,9 +151,10 @@ def wait_for_keypress(message = nil)
     end
 
     def current_connector
+      connector_settings = Core::ConnectorSettings.fetch(App::Config['connector_package_id'])
```

Comment:
Need to rename to `connector_id` after https://github.com/elastic/connectors-ruby/pull/194 is merged.